tests/data/MerchFactory.ts

```diff
@@ -65,6 +87,14 @@ export class MerchFactory {
     });
   }
 
+  public static fakeOptionMetadata(): MerchItemOptionMetadata {
+    return {
+      type: faker.datatype.hexaDecimal(10),
+      value: faker.datatype.hexaDecimal(10),
+      position: FactoryUtils.getRandomNumber(1, 10),
```

Comment:
If we always choose a random number for `position`, wouldn't we end up creating options with the same positions?

Reply:
I was thinking `position` could be a field that's solely managed by the frontend but stored in the backend, so that the frontend can order/display items however they like based on the position. Do you think we should be enforcing position order (i.e. it's always 0, 1, 2, 3, etc) and uniqueness (i.e. only one 1, only one 2, etc)?

Reply:
I initially thought so since the database should always have "clean"/valid data, but now I don't think so since FEs can order by `position` even if there're gaps and duplicate numbers.